src/components/strategies/common/InputBudget.tsx

```diff
@@ -145,7 +150,9 @@ export const InputBudget: FC<Props> = (props) => {
         </div>
         <div className="text-12 font-weight-500 flex min-h-[16px] flex-wrap items-center justify-between gap-10 font-mono">
           <p className="flex items-center gap-5 break-all text-white/60">
-            {fiatValue.gt(0) && prettifyNumber(fiatValue, { currentCurrency })}
+            {showFiatValue && prettifyNumber(fiatValue, { currentCurrency })}
+            {!hasFiatValue() &&
+              `Current ${selectedFiatCurrency} value unavailable`}
```

Comment:
Is there a case when you can have both ? If so I would rather have a ternary
Is there a case when you can have none ? If so what should be displayed ?

In any case I would rather move the output text as a variable above. I like to reduce code logic inside the template.

Reply:
You're right, done, it's much clearer now